activerecord/test/models/company.rb

```diff
@@ -13,7 +13,7 @@ class Company < AbstractCompany
   has_many :accounts
 
   scope :of_first_firm, lambda {
-    joins(:account => :firm).
+    joins(:accounts => :firm).
     where('firms.id' => 1)
   }
```

Comment:
Not sure if this was the original intent and a simple typo, or if the association for the `has_one :account` is failing to be inherited.